deploy.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
```

Comment:
Is there a particular reason why this is a bash script?

I feel that a Python script would be more readable for most users. I would expect that the environment that this script will be run on (either CI or a developer's machine) would have Python install.